gateway/tests/api/test_v1_program.py

```diff
@@ -414,3 +414,16 @@ def test_get_by_title(self):
             format="json",
         )
         self.assertEqual(programs_response_do_not_have_access.status_code, 404)
+
+    def test_get_jobs(self):
+        """Tests run existing authorized."""
+
+        user = models.User.objects.get(username="test_user_2")
+        self.client.force_authenticate(user=user)
+
+        response = self.client.get(
+            "/api/v1/programs/1a7947f9-6ae8-4e3d-ac1e-e7d608deec82/get_jobs/",
+            format="json",
+        )
+        print(response.data)
```

Comment:
Probably here instead to print I would check if it returns the expected number of jobs

Reply:
It was for debugging.  I'll take it out.  Thanks!